Riot/Modules/Common/ActivityIndicator/UserIndicatorPresenterWrapper.swift

```diff
@@ -41,10 +41,12 @@ import CommonKit
             return
         }
 
+        MXLog.debug("[UserIndicatorPresenterWrapper] Present loading indicator")
```

Comment:
```suggestion
        MXLog.debug("[UserIndicatorPresenterWrapper] Present activity indicator")
```

Reply:
I think `activity` does not belong here, because that name is currently only associated with `UIActivityIndicator`, which this system is not using. If anything I can probably rename the `presentActivityIndicator` methods.